spec/controllers/fee_categories_controller_spec.rb

```diff
@@ -54,8 +57,13 @@
   end
 
   describe "GET show" do
-    let(:category) { FeeCategory.create!(title: 'General Applications',
-                                         statutory_instrument_id: 1) }
+    let(:category) do
+      FeeCategory.create!(title: 'General Applications',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.